processor/transformprocessor/README.md

```diff
@@ -218,6 +218,30 @@ Examples:
 
 - `convert_gauge_to_sum("delta", true)`
 
+### extract_sum_metric
+
+> [!NOTE]  
+> This function supports Histograms, ExponentialHistograms and Summaries.
+
+`extract_sum_metric(is_monotonic, aggregation_temporality)`
+
+The `extract_sum_metric` function creates a new Sum metric from a Histogram, ExponentialHistogram or Summary's sum value. If the sum value of a Histogram or ExponentialHistogram data point is missing, no data point is added to the output metric. A metric will only be created if there is at least one data point.
+
+`is_monotonic` is a boolean representing the monotonicity of the new metric.
+
+The name for the new metric will be `<original metric name>_sum`. The fields that are copied are: `timestamp`, `starttimestamp`, `attibutes`, and `description` `aggregation_temporality`. As metrics of type Summary don't have an `aggregation_temporality` attribute, this attribute will be set to `AGGREGATION_TEMPORALITY_CUMULATIVE` for those metrics.
+
+The new metric that is created will be passed to all functions in the metrics statements list.  Function conditions will apply.
```

Comment:
```suggestion
The new metric that is created will be passed to all subsequent statements in the metrics statements list.
```
I think the last sentence should be a given since both Editors and conditions are available in any statement.